pkg/common-controller/framework_test.go

```diff
@@ -185,6 +202,35 @@ func isTestError(err error) bool {
 	return ok
 }
 
+func withClaimLabels(pvcs []*v1.PersistentVolumeClaim, labels map[string]string) []*v1.PersistentVolumeClaim {
+	for i := range pvcs {
+		if pvcs[i].ObjectMeta.Labels == nil {
+			pvcs[i].ObjectMeta.Labels = make(map[string]string)
+		}
+		for k, v := range labels {
+			pvcs[i].ObjectMeta.Labels[k] = v
+		}
+	}
+	return pvcs
+}
+
+func withVolumesCSIDriverName(pvs []*v1.PersistentVolume, driverName string) []*v1.PersistentVolume {
+	for i := range pvs {
+		pvs[i].Spec.CSI.Driver = driverName
```

Comment:
.CSI can be nil as well, can we need to handle that as well

Reply:
Yes! Good catch, thank you.